subsys/net/lib/mqtt/mqtt.c

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define LOG_MODULE_NAME net_mqtt
-#define NET_LOG_LEVEL LOG_LEVEL_ERR
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_mqtt, LOG_LEVEL_ERR);
```

Comment:
there is `CONFIG_MQTT_LOG_LEVEL` available.

Reply:
That symbol is only for new MQTT impl. This dir is for the legacy library which does not have log level defined.